source/_integrations/modbus.markdown

```diff
@@ -682,8 +682,18 @@ climates:
       description: "Address of On/Off state.
         When zero is read from this register, the HVAC state is set to Off, otherwise the `hvac_mode_register`
```

Comment:
The same here. 0 is the default, but the value could be different after your PR

Reply:
Done